src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -12,6 +12,7 @@ Cypress.Commands.add('fillVaTextInput', (field, value) => {
       .shadow()
       .find('input')
       .as('currentElement')
+      .clear({ force: true })
```

Comment:
^ This clear cmd adds resilience to flakiness from fields still being disabled when this method starts.